web/src/main/resources/application.yml

```diff
@@ -54,6 +54,8 @@ spring:
   jackson:
     serialization:
       write-dates-as-timestamps: true
+    deserialization:
```

Comment:
What is the meaning of this?

Reply:
It is mean that jackson will skip some unknown properties on requets (without exceptions)